discovery_engine_core/ai/kpe/src/tokenizer/encoding.rs

```diff
@@ -130,7 +120,6 @@ impl<const KEY_PHRASE_SIZE: usize> Tokenizer<KEY_PHRASE_SIZE> {
             valid_mask,
             active_mask,
         };
-        debug_assert!(encoding.is_valid(self.tokenizer.vocab_size(), KEY_PHRASE_SIZE));
```

Comment:
this assertion actually includes a handled error for the `valid_mask` sub-check, which is handled in the CNN. but since we run the tests with a debug build, we get a panic and hence a canceled future when running the dart tests. each of the validity checks are done in the respective Bert/CNN/Classifier modules anyways, so we don't loose anything by removing this assertion.